package.json

```diff
@@ -26,6 +26,7 @@
   },
   "dependencies": {
     "@auth0/auth0-react": "^2.2.1",
+    "@bpmn-io/form-js": "^1.6.1",
```

Comment:
You forgot to commit the lockfile, I pushed the fix to your branch

Reply:
Thank you!